Resources/config/services.yml

```diff
@@ -2,6 +2,7 @@ services:
     opale.kernel.listener.command_dispatch:
         class: Opale\ConsoleExceptionBundle\Listener\ConsoleExceptionListener
         arguments:
-            logger: "@logger"
+            - "@logger"
+        deprecated: 'The "%service_id%" service is deprecated and useless starting with Symfony 3.3'
```

Comment:
This is supported in recent versions of sf only, and should be added only if the user has sf >3.3 (so you should probably do it in php, in your extension)